aspnet/mvc/controllers/areas.rst

```diff
@@ -60,11 +78,17 @@ Once you've defined the folder hierarchy, you need to tell MVC that each control
       [Area("Products")]
       public class HomeController : Controller
       {
-          // GET: /<controller>/
+          // GET: /Products/Home/Index
```

Comment:
The home controller is invoked like a products controller? Is that right?

Can you put a good simple sample of areas on github and I'll use your sample an literal include. We don't allow this much copy/paste. I can't figure out what's going on for sure and customers won't be able to either. Once we have a project in github I can rewrite this doc and customers will be able to download the project an experiment with it.